core/helpers.py

```diff
@@ -17,6 +18,18 @@ def build_social_link(template, request, title):
     )
 
 
+def cms_component_supports_activated_language(activated_language, languages):
+    for language in languages:
```

Comment:
we can unpack the `languages` list like so:

```
    for language_code, label in languages:
        if language_code == activated_language:
            return True

```
when unpacking a nice convention is to use `_` for the "thrown away" variable, and we can, and we can use `any` for terseness:

```
return any(code == activated_language for code, _ in languages)
```
quite pretty :)

Reply:
whaaaaaaaat that's neat. i love python

Reply:
:heart: